libs/resolver/loader/module_loader.rs

```diff
@@ -523,22 +523,17 @@ impl<TSys: ModuleLoaderSys> PreparedModuleLoader<TSys> {
         specifier: Cow::Borrowed(specifier),
         media_type: MediaType::Wasm,
       }))),
-      Some(deno_graph::Module::External(module))
-        if matches!(
-          requested_module_type,
-          RequestedModuleType::Bytes | RequestedModuleType::Text
-        ) =>
-      {
+      Some(deno_graph::Module::External(module)) => {
+        if module.specifier.as_str().contains("/node_modules/") {
+          return Ok(None);
+        }
         Ok(Some(CodeOrDeferredEmit::ExternalAsset {
           specifier: &module.specifier,
         }))
       }
```

Comment:
This is iffy, but this code needs to return `Ok(None)` for byonm files and pass through raw and source-only imports.

Reply:
Seems a bit hacky, but let's address this in a follow up